src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -5,9 +5,8 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.address.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.testutil.TypicalPersons.CARL;
+import static seedu.address.testutil.TypicalPersons.BENSON;
```

Comment:
I am not sure if it is recommended to remove test stubs as less stubs may lead to edge cases not considered. KIV perhaps add more stubs in the future.